WalletWasabi/WebClients/ExchangeRateProviders.cs

```diff
@@ -7,41 +9,43 @@
 using WalletWasabi.Logging;
 using WalletWasabi.WebClients.BlockchainInfo;
 using WalletWasabi.WebClients.Coinbase;
+using WalletWasabi.WebClients.Bitstamp;
+using WalletWasabi.WebClients.CoinGecko;
 using WalletWasabi.WebClients.Gemini;
 using WalletWasabi.WebClients.ItBit;
 using WalletWasabi.WebClients.SmartBit;
+using System.Linq;
 
 namespace WalletWasabi.WebClients
 {
 	public class ExchangeRateProvider : IExchangeRateProvider
 	{
 		private readonly IExchangeRateProvider[] ExchangeRateProviders =
 		{
-			new SmartBitExchangeRateProvider(new SmartBitClient(Network.Main)),
 			new BlockchainInfoExchangeRateProvider(),
+			new BitstampExchangeRateProvider(),
+			new CoinGeckoExchangeRateProvider(),
 			new CoinbaseExchangeRateProvider(),
 			new GeminiExchangeRateProvider(),
-			new ItBitExchangeRateProvider()
+			new ItBitExchangeRateProvider(),
+			new SmartBitExchangeRateProvider(new SmartBitClient(Network.Main))
 		};
 
 		public async Task<List<ExchangeRate>> GetExchangeRateAsync()
 		{
-			List<ExchangeRate> exchangeRates = null;
-
 			foreach (var provider in ExchangeRateProviders)
 			{
 				try
 				{
-					exchangeRates = await provider.GetExchangeRateAsync();
-					break;
+					return await provider.GetExchangeRateAsync();
 				}
 				catch (Exception ex)
 				{
 					// Ignore it and try with the next one
 					Logger.LogTrace(ex);
 				}
 			}
-			return exchangeRates;
+			return Enumerable.Empty<ExchangeRate>().ToList();
```

Comment:
could we make this a const somewhere? this could make some unnecessary allocs i think